packages/terra-search-field/lib/SearchField.scss

```diff
@@ -0,0 +1,45 @@
+@import '~terra-mixins';
+
+.terra-SearchField {
+  display: inline-flex;
+
+  .terra-SearchField-input {
+    @include terra-border-radius-start(0.286em);
+    @include terra-border-radius-end(0);
+
+    appearance: textfield;
+    border: 0.071em solid $terra-grey-15;
+    border-right: 0;
```

Comment:
Here is an example of one place it is used:
https://github.com/cerner/terra-core/blob/d7e750f7dbc1b4b409e1a4ea08e5d8064db0ee36/packages/terra-progress-bar/src/ProgressBar.scss#L35


```
@include terra-border-end(0);
```